app/decorators/cells/decidim/assemblies/content_blocks/highlighted_assemblies_cell_decorator.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# This decorator filter assemblies by permissions in home content block.
+Decidim::Assemblies::ContentBlocks::HighlightedAssembliesCell.class_eval do
+  include FilterParticipatorySpacesHelper
+
+  def highlighted_assemblies
+    @highlighted_assemblies ||= if current_user.admin?
+                                  Decidim::Assemblies::OrganizationPrioritizedAssemblies
```

Comment:
Usually if we want the standard behavior we delegate to the decorated method. Check this example: https://github.com/gencat/participa-catalunya/blob/master/app/decorators/decidim/courses/controllers/admin/courses_controller_decorator.rb